index.js

```diff
@@ -82,6 +56,8 @@ module.exports = {
     // inside `onPreBuild`.
     const nextOnNetlify = require('next-on-netlify')
     nextOnNetlify({ functionsDir: FUNCTIONS_SRC, publishDir: PUBLISH_DIR })
+
+    return true
```

Comment:
Build plugin methods should not return values. Returning values is reserved for future features.

Reply:
This is done per my suggestion to be able to test that the method skips artifacts generation based on return value.
Another option is indeed to verify that nothing was generated in the publish/functions directory (or remove those tests completely)

Reply:
It seems to me checking for the side effect of the function would give a stronger guarantee that it works as intended.

The problem with plugins returning values in methods is that they would prevent us from using return values in future features, which we might want to do.